src/components/__tests__/LibrariesListItem-test.tsx

```diff
@@ -58,11 +67,11 @@ describe("LibrariesListItem", () => {
       expect(wrapper.state().color).to.equal("warning");
       expect(wrapper.find(".panel-warning").length).to.equal(1);
 
-      (wrapper.instance() as any).updateColor("cancelled", "production");
+      (wrapper.instance() as any).updateColor(["cancelled", "production"]);
```

Comment:
Which one is registry stage and library stage? It'll be clearer to have them as variable names so you can see which param is which stage.

Reply:
Does it matter which is which, though?  The color-coding logic is just based on the combination of the two stages--it checks 1) whether either stage is cancelled, and 2) whether both stages are in production.  The individual values should only matter for `LibraryDetailPage` and its children (i.e. for populating and submitting the form); if `LibrariesListItem` suddenly cares about which is which, then something has gone wrong...